closure/compiler/closure_js_binary.bzl

```diff
@@ -67,12 +68,20 @@ def _impl(ctx):
     args += ["--debug"]
   elif is_using_closure_library(srcs):
     args += ["--define=goog.DEBUG=false"]
+  if is_using_closure_library(srcs) and language_out in ["ECMASCRIPT6_TYPED",
+                                                         "ECMASCRIPT6_STRICT",
+                                                         "ECMASCRIPT5_STRICT"]:
+    args += ["--define=goog.STRICT_MODE_COMPATIBLE"]
+  if contains_file(srcs, ctx.attr._soyutils_usegoog):
+    args += ["--define=goog.soy.REQUIRE_STRICT_AUTOESCAPE"]
   for entry_point in ctx.attr.entry_points:
     _validate_entry_point(entry_point, srcs)
     args += ["--entry_point=" + entry_point]
   if ctx.attr.pedantic:
     args += JS_PEDANTIC_ARGS
     args += ["--use_types_for_optimization"]
+  if ctx.attr.output_wrapper:
+    args += ["--output_wrapper=\"%s\"" % ctx.attr.output_wrapper]
```

Comment:
Is it safe to pass this without the quotes? Can we have a file_test in `closure/compiler/test/` somewhere that tests an output_wrapper witha a space?